build.cmd

```diff
@@ -51,7 +51,6 @@ set __SkipConfigure=
 set __SkipMscorlibBuild=
 set __SkipNativeBuild=
 set __SkipTestBuild=
-set __DoCrossgen=
```

Comment:
It actually does nothing when passed to build.cmd. I will remove instances from netci.groovy